aws/v11.5.2/release.diff

```diff
@@ -0,0 +1,63 @@
+  - name: chart-operator                                             - name: chart-operator
+    version: 0.12.4                                                    version: 0.12.4
+  - componentVersion: 1.16.5                                         - componentVersion: 1.16.5
+    name: cluster-autoscaler                                           name: cluster-autoscaler
+    version: 1.16.0                                                    version: 1.16.0
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.2.0                                                     version: 1.2.0
+  - componentVersion: 0.7.2                                          - componentVersion: 0.7.2
+    name: external-dns                                                 name: external-dns
+    version: 1.2.2                                                     version: 1.2.2
+  - componentVersion: 3.5.0                                          - componentVersion: 3.5.0
+    name: kiam                                                         name: kiam
+    version: 1.3.0                                                     version: 1.3.0
+  - componentVersion: 1.9.5                                          - componentVersion: 1.9.5
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.1.0                                                     version: 1.1.0
+  - componentVersion: 0.3.3                                          - componentVersion: 0.3.3
+    name: metrics-server                                               name: metrics-server
+    version: 1.1.0                                                     version: 1.1.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.9.0                                                     version: 1.9.0
+  - componentVersion: 0.18.1                                         - componentVersion: 0.18.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.0                                                     version: 1.6.0
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                 |      version: 8.7.6
+    version: 8.7.5                                              <
```

Comment:
I am a bit confused what `|` and `<` means.

Reply:
The diff creates the impression that `releaseOperatorDeploy: true` is now not defined anymore, but in fact it is.

Reply:
this is generated by `devctl`, so I don't really know

Reply:
Maybe @tfussell can enlighten us. 😬

Reply:
`|` means there's a difference on those two lines, `<` means the line is only on the left side. Please update `devctl` with [this PR](https://github.com/giantswarm/devctl/pull/87) and run this again to make sure you get `releaseOperatorDeploy: true` in your release.

Reply:
diff got updated